src/Avalonia.Base/Data/InstancedBinding.cs

```diff
@@ -14,39 +15,49 @@ namespace Avalonia.Data
     /// </remarks>
     public class InstancedBinding
     {
+        private Subject<Unit>? _explicitSourceUpdateSubject;
```

Comment:
I think that you can use `SingleSubscriberObservableBase<T>` here since there should be only one subscriber from what I understand.